plugins/pipelines/create.js

```diff
@@ -113,6 +121,43 @@ module.exports = () => ({
 
                                                 return defaultCollection.update();
                                             })
+                                            .then(() => {
+                                                if (autoKeysGeneration) {
+                                                    return pipelineFactory.scm
+                                                        .addDeployKey({
+                                                            scmContext,
+                                                            checkoutUrl,
+                                                            token: pipelineToken
+                                                        })
+                                                        .then(privateDepKey => {
```

Comment:
should add error handling. scm's which have not implemented `_addDeployKey` throws error. 

catch log and respond error part of API payload, so that UI can display.